addons/mail/static/src/messaging/entity/composer/composer.js

```diff
@@ -258,6 +364,12 @@ function ComposerFactory({ Entity }) {
         thread: one2one('Thread', {
             inverse: 'composer',
         }),
+        mentionedPartners: one2many('Partner', {
+            inverse: 'composerMentionedPartners',
```

Comment:
For information, `inverse` relation is now optional, if you don't use it you can remove it (both the "inverse" param here, and the whole definition on the other model). (I didn't check if it was used or not, to double check and do if so).